js/serial_backend.js

```diff
@@ -443,6 +443,18 @@ function have_sensor(sensors_detected, sensor_code) {
         case 'sonar':
             return bit_check(sensors_detected, 4);
     }
+
+    if (bit_check(sensors_detected, 5)) {
```

Comment:
I think it makes more sense to gate this on apiVersion, since at this point we know that firmware prior to 1.36 (I think) doesn't report on the gyro, firmware post it does.